Automattic/wp-calypso
# Jetpack Checkout: submit website URL to WPCOM endpoint

#### Changes proposed in this Pull Request

* Make the `Your website address:` input box located in the post-purchase Thank You page submit the URL to WPCOM.

#### Testing instructions

* Patch your sandbox with D63168-code.
* Patch your sandbox with D63915-code.
* Sandbox `public-api.wordpress.com`.
* Complete the site-less checkout flow.
* On the Thank You page, enter a URL.
* Verify that the URL was appended to the ticket as a private note.

Related to 1200479326344990-as-1200571786740545
